bin/eg-generator.js

```diff
@@ -8,7 +8,14 @@ module.exports = class EgGenerator extends Generator {
     this._configuration = null;
     this.eg = this.env.eg;
     this.argv = this.env.argv;
-    this.admin = require('../lib/admin')(config.gatewayConfig.admin);
+    if (this.argv) {
+      let cliConfig = config.systemConfig.cli || {};
```

Comment:
I'll add EG_ADMIN_URL, but we should not add `config.gatewayConfig.admin`
'cuz config may be different, for example if admin API is protected by EG.
Also `config.cli` has additional property `protocol`, that doesn't make sense for REST admin